docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -55,25 +55,28 @@ If Camunda 8 Run fails to start, run the [shutdown script](#shut-down-camunda-8-
+| `--config <path>`          | Applies the specified Zeebe [`application.yaml`](/self-managed/zeebe-deployment/configuration/configuration.md).                                                                                                                              |
+| `--username <arg>`         | Configure the first user’s username to `<arg>`.                                                                                                                                                                                               |
+| `--password <arg>`         | Configure the first user’s password to `<arg>`.                                                                                                                                                                                               |
+| `--keystore <arg>`         | Configure the TLS certificate for HTTPS. If not specified, use HTTP. For more information, see [enabling TLS](#enable-tls).                                                                                                                   |
+| `--keystorePassword <arg>` | Provide the password to use with a JKS keystore file.                                                                                                                                                                                         |
+| `--port <arg>`             | Configure the Camunda core port to the value provided (default: 8080).                                                                                                                                                                        |
+| `--log-level <arg>`        | Set a different log level for the Camunda core.                                                                                                                                                                                               |
+| `--docker`                 | Download and run the Camunda Docker Compose distribution. Any additional options are not supported at this time, and will be ignored. See the [shutdown script](#shut-down-camunda-8-run) for information on stopping the Docker application. |
+| `--disable-elasticsearch`  | Do not start the built-in Elasticsearch. Ensure another Elasticsearch instance is provided via `--config`. See the [external Elasticsearch](#start-external-elasticsearch) options for more information.                                      |
 
 ## Work with Camunda 8 Run
 
 ### Access Camunda components
 
 All Camunda 8 Run components can be accessed using the username/password combination `demo`/`demo`.
 
+:::note
+The URLs for the Docker Compose application can be found in the [Docker Compose](/self-managed/setup/deploy/local/docker-compose.md#access-components) documentation.
+:::
```

Comment:
@akeller Opinions on making this a tabbed experience to list both, vs just linking to the Docker docs? If using tabs, I wasn't sure if "Java" as a tab header for the default experience would be clear enough to every user that that's what they needed.

Reply:
@conceptualshark let's just link for now. I'd like to get a more comprehensive understanding of the E2E journey for C8 Run, and specifically the onboarding experience. (See [here](https://camunda.slack.com/archives/C03UR0V2R2M/p1743781745857859) and [here](https://camunda.slack.com/archives/C089AUH6065/p1743771189872659))